packages/lesswrong/server/callbacks/commentCallbacks.ts

```diff
@@ -1,667 +1,141 @@
-  let messageContents = `
-  <p>Unfortunately, I rejected your ${rejectedContentLink}.</p>
-  <p>LessWrong aims for particularly high quality (and somewhat oddly-specific) discussion quality. We get a lot of content from new users and sadly can't give detailed feedback on every piece we reject, but I generally recommend checking out our <a href="https://www.lesswrong.com/posts/LbbrnRvc9QwjJeics/new-user-s-guide-to-lesswrong">New User's Guide</a>, in particular the section on <a href="https://www.lesswrong.com/posts/LbbrnRvc9QwjJeics/new-user-s-guide-to-lesswrong#How_to_ensure_your_first_post_or_comment_is_well_received">how to ensure your content is approved</a>.</p>`
-  if (rejectedReason) {
-    messageContents += `<p>Your content didn't meet the bar for at least the following reason(s):</p>
-    <p>${rejectedReason}</p>`;
-  }
-  return messageContents;
+async function commentCreateAsync(props: AfterCreateCallbackProperties<'Comments'>) {
+  await newCommentTriggerReview(props);
+  await trackCommentRateLimitHit(props);
+  await checkModGPTOnCommentCreate(props);
+
+  // Elastic callback
 }
 
-async function commentsRejectSendPMAsync (comment: DbComment, currentUser: DbUser) {
-  let rejectedContentLink = "[Error: content not found]"
-  let contentTitle: string|null = null
-
-  if (comment.tagId) {
-    const tag = await Tags.findOne(comment.tagId)
-    if (tag) {
-      contentTitle = tag.name
-      rejectedContentLink = `<a href=${tagGetDiscussionUrl({slug: tag.slug}, true)}` + `?commentId=${comment._id}">comment on ${tag.name}</a>`
-    }
-  } else if (comment.postId) {
-    const post = await Posts.findOne(comment.postId)
-    if (post) {
-      contentTitle = post.title
-      rejectedContentLink = `<a href="https://lesswrong.com/posts/${post._id}/${post.slug}?commentId=${comment._id}">comment on ${post.title}</a>`
-    }
-  }
-
-  const commentUser = await Users.findOne({_id: comment.userId})
-
-  let messageContents = getRejectionMessage(rejectedContentLink, comment.rejectedReason)
-  
-  // EAForum always sends an email when deleting comments. Other ForumMagnum sites send emails if the user has been approved, but not otherwise (so that admins can reject comments by mediocre users without sending them an email notification that might draw their attention back to the site.)
-  const noEmail = isEAForum 
-  ? false 
-  : !(!!commentUser?.reviewedByUserId && !commentUser.snoozedUntilContentCount)
-
-  await sendModerationPM({
-    action: 'rejected',
-    comment,
-    messageContents,
-    lwAccount: currentUser,
-    noEmail,
-    contentTitle
-  });
+async function commentNewAsync(comment: DbComment, currentUser: DbUser | null, collection: CollectionBase<'Comments'>, props: AfterCreateCallbackProperties<'Comments'>) {
+  // I'm pretty sure this is just a strict subset of the operations in commentsNewOperations
+  // await alignmentCommentsNewOperations(comment, props.context);
```

Comment:
This is one of the few things I'm at all confused about, and I'm pretty sure we just had a redundant callback here for a few years.